pkg/ocm/clusters.go

```diff
@@ -396,6 +396,22 @@ func (c *Client) HasAClusterUsingOidcConfig(bucketName string) (bool, error) {
 	return false, nil
 }
 
+func (c *Client) HasAClusterUsingOidcProvider(oidcEndpointURL string) (bool, error) {
```

Comment:
Maybe it is worth returning the cluster's ID?
So we can display the ID in the error message, to provide more info to the user.

Reply:
We currently don't have any statistics into how many clusters client's will use in a single oidc. So I thought about leaving as is for now as having an idea of how many pages we'll need to fetch before adding this UX feature